photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinate.scala

```diff
@@ -190,7 +192,12 @@ object RandomEffectCoordinate {
     //TODO: Need a better design that properly unpersists the broadcasted variables and persists the computed RDD
     val modelsForPassiveDataBroadcast = passiveData.sparkContext.broadcast(modelsForPassiveData)
     val passiveScores = passiveData.mapValues { case (randomEffectId, labeledPoint) =>
-      modelsForPassiveDataBroadcast.value(randomEffectId).computeScore(labeledPoint.features)
+      ScoredGameDatum(
```

Comment:
We could have a `ScoreGameDatum.apply` in the object that takes a labeled point and supplies the default `Map()` at the end.